submissions/5mountains/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,115 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Resident {
+  constructor(species, name, gender) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+  }
+  toString() {
+    return `Wonderful creature - <b>${
+      this.species
+    }</b>, whose name is <b>${
+      this.name
+    }</b> gender is <b>${
+      this.gender
+    }</b>!`;
+  }
+}
+
+class Human extends Resident {
+  constructor(name, gender, saying, friends = 'no one') {
+    super('human', name, gender);
+    this.legs = 2;
+    this.hands = 2;
+    this.saying = saying;
+    this.friends = friends;

```

Comment:
Looks like `friends` is a property shared by all inhabitants. Worth having it and handle it at the super class level.

Reply:
Done